src/styles/stars.scss

```diff
@@ -0,0 +1,35 @@
+@import './variables';
```

Comment:
Ensure that the `variables` file contains definitions for `$star-size` and `$star-margin`. These variables are crucial for the proper styling of the stars.